packages/headless/src/features/insight-search/insight-search-request.ts

```diff
@@ -45,6 +45,15 @@ export const buildInsightBaseRequest = (
       parentField: state.folding.fields.child,
       filterFieldRange: state.folding.filterFieldRange,
     }),
+    ...(state.generatedAnswer && {
+      pipelineRuleParameters: {
+        mlGenerativeQuestionAnswering: {
+          responseFormat: state.generatedAnswer.responseFormat,
+          citationsFieldToInclude:
```

Comment:
Is this something we want to send in the request of the search useCase as well? I don't currently see `citationsFieldToInclude` included in the current search-request

Reply:
They are there: https://github.com/coveo/ui-kit/blob/bd95c9fbad8a9f33b14cc6448607069146a20c4c/packages/headless/src/features/search/search-request.ts#L77